static/sass/print.scss

```diff
@@ -23,18 +23,21 @@ body {
   color: $color-x-dark;
   font-family: "Ubuntu", "Segoe UI", "Roboto", "Oxygen", "Cantarell",
     "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
-  line-height: 1.3;
+  line-height: 1.5em;
 }
 
 a,
 blockquote,
-table,
 pre,
 [class^="p-strip"],
 [class*=" p-strip"] {
   page-break-inside: avoid;
 }
 
+table {
+  margin: 1.5em 0;
```

Comment:
We use `rem` instead of `em` across all of our codebases to keep our spacing consistent. You can read more about the difference between them here: https://webdesign.tutsplus.com/tutorials/comprehensive-guide-when-to-use-em-vs-rem--cms-23984